safe-global/safe-wallet-web
# fix: Show Safe activation status in safe list

## What it solves

Resolves #3492 

## How this PR fixes it

Displays a Chip in the Safe List either saying "Not activated" or "Activating account" depending on the deployment status of a Safe

## How to test it

1. Create a counterfactual safe
2. Open the Sidebar and observe a new Chip saying "Not activated"
3. Activate the Safe
4. Open the Sidebar and observe the same Chip saying "Activating account" with a different color and loading spinner
5. The same should be visible on the login page
6. Already deployed safes should not be affected by this
7. Watchlist safes should not be affected by this

## Screenshots

<img width="647" alt="Screenshot 2024-03-28 at 11 10 35" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/6bab3774-84cb-425e-ba38-f21385495c39">
<img width="471" alt="Screenshot 2024-03-28 at 11 11 38" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/ef442fe1-444b-4e11-bb06-5ac70e289275">


## Checklist
